subworkflows/local/utils_nfcore_phaseimpute_pipeline/main.nf

```diff
@@ -210,27 +210,22 @@ workflow PIPELINE_INITIALISATION {
     //
     // Create posfile channel
     //
-
     if (params.posfile) {
         ch_posfile = Channel
             .fromSamplesheet("posfile")
             .map {meta, file -> [ meta, file ]}
     } else {
-        ch_posfile = [[[],[]]]
+        ch_posfile = [[],[]]
     }
 
     //
     // Create chunks channel
     //
-
     if (params.chunks) {
         ch_chunks = Channel
-        .fromSamplesheet("chunks")
-        .map {
-            meta, file ->
-                [ meta, file ]
-    }} else {
-        ch_chunks = [[[],[]]]
+            .fromSamplesheet("chunks")
```

Comment:
I am not sure if the 
```
meta, file -> [ meta, file ]
```
should be removed?  I am going to open a new issue to add a test for this just to be sure.